docker-compose-tests.yml

```diff
@@ -25,7 +25,7 @@ services:
       - gp-finder-travis-tests-node_modules:/code/node_modules
     environment:
       COVERALLS_REPO_TOKEN: ${COVERALLS_REPO_TOKEN}
-      NODE_ENV: development
+      NODE_ENV: ${NODE_ENV-development}
```

Comment:
To keeo it compatible with this: https://docs.rancher.com/rancher/v1.5/en/cli/variable-interpolation/#variable-interpolation-formats and my understanding I would say stick to only unset. I think I need to understand how the envs are used more if we were to have a `start-prod` /cc @adamgraves-choices